app/src/main/java/io/github/droidkaigi/confsched/util/DataBindingAttributeUtil.java

```diff
@@ -84,4 +89,26 @@ public static void setSessionFab(FloatingActionButton fab, @NonNull Session sess
         fab.setSelected(session.checked);
     }
 
+    @BindingAdapter("forceRtlDirection")
+    public static void setForceRtlDirection(FlowLayout flowLayout, boolean isCenterVertical) {
+        if (LocaleUtil.shouldRtl(flowLayout.getContext())) {
+            ViewCompat.setLayoutDirection(flowLayout, ViewCompat.LAYOUT_DIRECTION_RTL);
+
+            int gravity = isCenterVertical ? GravityCompat.START | Gravity.CENTER_VERTICAL : GravityCompat.START;
+            flowLayout.setGravity(gravity);
+        }
+    }
+
+    @BindingAdapter("textRtlConsidered")
+    public static void setTextRtlConsidered(TextView textView, String text) {
+        textView.setText(LocaleUtil.getRtlConsideredText(text, textView.getContext()));
```

Comment:
When text first charactor is not Arabic, RTL is ignored. Like this.

<img width="336" alt="2016-02-08_18_19_46_png" src="https://cloud.githubusercontent.com/assets/1269214/12938703/6f086256-cffa-11e5-8867-1b14891a779b.png">

One solution is to insert `RTL Mark`.
http://www.fileformat.info/info/unicode/char/200f/index.htm

If you use `app:textRtlConsidered` instead of `android:text`, RTL Mark is inserted at the top of text.